### PR TITLE
RDK-39119: Implement LED control

### PR DESCRIPTION
**Pull request by acheri988**
Reason for change:
Implement LED control
Test Procedure: None
Risks: Low

Change-Id: I65ec43ee44c9d62407ca2082e69a9609dc9d6aef Signed-off-by:Anooj Cheriyan <Anooj_Cheriyan@comcast.com>

### CODE REVIEW

File: LEDControl/Module.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+#pragma once
+#ifndef MODULE_NAME
+#define MODULE_NAME Plugin_FrontPanel
```
Comment (apatel859) [inline LEDControl/Module.h:22]: rename FrontPanel

File: LEDControl/LEDControlPlugin.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://raw.githubusercontent.com/rdkcentral/rdkservices/main/Tools/json_generator/schemas/plugin.schema.json",
+    "info": {
+      "title": "LEDControl Plugin",
+      "callsign": "org.rdk.LEDControl",
+      "locator": "libWPEFrameworkLEDControl.so",
+      "status": "production",
+      "description": "The `LEDControl` plugin allows you to control brightness and power state for LEDs on a set-top box."
```
Comment (apatel859) [inline LEDControl/LEDControlPlugin.json:8]: Say device not STB and this will run on all rdk devices

File: LEDControl/LEDControl.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                    }
+                },
+                "required": [
+                    "state",
+                    "success"
+                ]
+            }
+        },
+        "setLEDState":{
+            "summary": "Change the device led state to one menttioned in the agrument",
```
Comment (apatel859) [inline LEDControl/LEDControl.json:62]: typo

**Comment by acheri988** [comment]
New one created: https://github.com/rdkcentral/rdkservices/pull/3511

**Pull request closed without merging**